src/components/App.vue

```diff
@@ -102,8 +103,14 @@ export default {
       if (!url || this.downloading) return
 
       this.downloading = true
-      const res = await this.fetchImage(url)
-      download(res.url)
+      const file_name = `${res.author}-${res.media_id}.jpg`
+      const dl = await axios({
```

Comment:
1. I tried directly `download(url, filename)` but the downloaded file is not valid image.  
After see the [downloadjs](https://github.com/rndme/download) doc, i found get blob response data and then set filename and mime type worked.
2. delete `const res = await this.fetchImage(url)` by mistake